src/storage/lock_manager/lock_waiting_queue.rs

```diff
@@ -1205,4 +1235,44 @@ mod tests {
         queues.must_not_contain_key(b"k1");
         assert_eq!(queues.entry_count(), 0);
     }
+
+    #[bench]
+    fn bench_update_lock_wait_empty(b: &mut test::Bencher) {
+        let queues = LockWaitQueues::new(MockLockManager::new());
+        queues.mock_lock_wait(b"k1", 5, 6, false);
+
+        let mut lock_info = kvrpcpb::LockInfo::default();
+        let key = b"t\x00\x00\x00\x00\x00\x00\x00\x01_r\x00\x00\x00\x00\x00\x00\x00\x01";
+        lock_info.set_key(key.to_vec());
+        lock_info.set_primary_lock(key.to_vec());
+        lock_info.set_lock_version(10);
+        lock_info.set_lock_for_update_ts(10);
+        let lock_info = vec![lock_info];
+
+        b.iter(|| {
+            queues.update_lock_wait(lock_info.clone());
+        });
+    }
+
+    #[bench]
+    fn bench_update_lock_wait_queue_len_512(b: &mut test::Bencher) {
```

Comment:
Any performance difference for `lockWaitQueues.update_lock_wait` if the queue size is large?

Reply:
The result was pasted in the above comment. 

```
test storage::lock_manager::lock_waiting_queue::tests::bench_update_lock_wait_empty         ... bench:         216 ns/iter (+/- 7)
test storage::lock_manager::lock_waiting_queue::tests::bench_update_lock_wait_queue_len_512 ... bench:     133,810 ns/iter (+/- 5,278)
```